src/LightArea.h

```diff
@@ -39,7 +39,31 @@ class CLightArea : public CLightOmni
 	virtual std::optional<Vec3f>	illuminate(Ray& ray) override
 	{
 		// --- PUT YOUR CODE HERE ---
-		return std::nullopt;
+        Vec2f sample(0, 0);
+        size_t k = 0;
```

Comment:
it will be all the time 0
You need to make it static or member variable to save the state over the different calls of the function